regress/pesterTests/SSH.Tests.ps1

```diff
@@ -106,7 +106,12 @@ Describe "E2E scenarios for ssh client" -Tags "CI" {
 
         It "$tC.$tI - test version" {
             iex "cmd /c `"ssh -V 2> $stderrFile`""
-            $stderrFile | Should Contain "OpenSSH_"
+            $stderrFile | Should Contain "OpenSSH_for_Windows"
+        }
+
+        It "$tC.$tI - test version" {
```

Comment:
This is duplicate of  above testcase?

Reply:
hmm yes. Clueless what happened. I'll remove it.